README.md

```diff
@@ -3,9 +3,11 @@
 # Orange Studio Webpage
 
 > Capstone project for Microverse. A webpage for a web design studio. 
+> Check project [here](https://melaniesigrid.github.io/OrangeStudio/)
+> Video introduction of project [here](https://www.loom.com/share/a3255081058447c8b041fd1f26859241)
 
 
-![screenshot](images/app_screenshot.png)
+![screenshot](images/Orange-studio-screenshot.png)
 
 ## Authors
```

Comment:
- [ ] Don't forget to give appropriate credit to Cindy Shin who is the author of the original design on the acknowledgment section of your readme.